package/network/config/ltq-vdsl-app/files/10-xdsl_rename

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+if [ "$ACTION" = add ]; then
+	[ "$DEVICENAME" = "nas0" ] ||[ "$DEVICENAME" = "ptm0" ] || exit
+
+	ip link set $DEVICENAME name dsl0
```

Comment:
The ltq-vdsl-app package now needs to depend on ip

Reply:
does the busybox ip command not support this feature?

Reply:
Yes, the busybox ip command provides this feature:

```
root@LEDE:/# /bin/busybox ip link set wlan0 name mk1
[  197.621145] ath10k_pci 0000:02:00.0 mk1: renamed from wlan0
```
If I add a `DEPENDS:= +ip` to the package, ip-tiny is autoselected without any need. Seem to me the ip depend doesn't work as expected with the busybox applet.

Reply:
Busybox now provides "ip" (https://git.lede-project.org/?p=source.git;a=commit;h=1cec4d4ef0145b48094ec0df40c18b8fbb315cf0); specifying ip depend should not select ip-tiny anymore now